packer/studio-selfhosted.pkr.hcl

```diff
@@ -0,0 +1,129 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.0.0"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variables {
+  image_name             = "studio-selfhosted"
+  image_description      = "Iterative Studio Selfhosted - {{isotime `2006-01-02`}}"
+  aws_build_region       = "us-west-1"
+  aws_build_instance     = "m6i.large"
+  aws_build_ubuntu_image = "*ubuntu-*-22.04-amd64-server-*"
+  skip_create_ami        = true
+}
+
+locals {
+  aws_tags = {
+    ManagedBy   = "packer"
+    Name        = var.image_name
+    Environment = "prod"
+    BuildDate   = "{{isotime `2006-01-02`}}"
+  }
+
+  aws_release_regions = [
```

Comment:
Thoughts on enabling additional regions?

Reply:
I think we have it blocked somewhere - probably we need to enable other regions in console first..

Reply:
I think you're right. I see that the ones we have enabled are AWS defaults, so let's just keep them